src/algorithm/contains.rs

```diff
@@ -166,6 +175,7 @@ impl<T> Contains<Bbox<T>> for Bbox<T>
 mod test {
     use types::{Coordinate, Point, LineString, Polygon, MultiPolygon, Bbox};
     use algorithm::contains::Contains;
+        #[test]
```

Comment:
That was unintentional. I had temporarily made a test to verify it worked, then deleted it.